javaparser-core/src/main/java/com/github/javaparser/ParserConfiguration.java

```diff
@@ -287,43 +283,51 @@ public void process(ParseResult<? extends Node> result, ParserConfiguration conf
             }
         }
 
-        UnicodeEscapeProcessor unicodeProcessor = new UnicodeEscapeProcessor();
-        preProcessors.add(unicodeProcessor);
-        postProcessors.add(unicodeProcessor);
-
-        LineEndingProcessor lineEndingProcessor = new LineEndingProcessor();
-        preProcessors.add(lineEndingProcessor);
-        postProcessors.add(lineEndingProcessor);
+        processors.add(UnicodeEscapeProcessor::new);
```

Comment:
So it looks like this now. You don't add a processor, but you add a supplier that creates a processor. ::new is such a thing.